recipe/build.sh

```diff
@@ -24,9 +24,12 @@ for i in `ls`; do
                 # Shared and static libraries are symlinked in $PREFIX/lib
                 ln -sv ${PREFIX}/${targetsDir}/$j ${PREFIX}/$j
 
-                if [[ $j =~ \.so\. ]]; then
-                    patchelf --set-rpath '$ORIGIN' ${PREFIX}/${targetsDir}/$j
-                fi
+                # Conda-forge needs these .sos to find libraries in the environment, because
+                # the runtime dependencies are packaged in their ecosystem.
+                # For defaults, they're in the default system lib location.
+                # if [[ $j =~ \.so\. ]]; then
+                #     patchelf --set-rpath '$ORIGIN' ${PREFIX}/${targetsDir}/$j
```

Comment:
I think we should avoid diverging from conda-forge too much. Having `$ORIGIN` in `RUNPATH` doesn't hurt in this case. NVidia might actually have a reason to prefer `RUNPATH` over `RPATH` here (because conda-build will set `RPATH` to `$ORIGIN/.` automatically).

So if anything, we should wait to get an answer on https://github.com/conda-forge/cuda-cudart-feedstock/issues/21 before proceeding with this change.

Reply:
I'll wait for the answer from conda-forge

Reply:
From https://github.com/conda-forge/cuda-cudart-feedstock/issues/21 (and https://github.com/conda-forge/cuda-feedstock/issues/10), we need to keep this. We could create an issue on the cuda feedstock to suggest using `--force-rpath` which would silence conda-build.

Reply:
yeah. This would also change precedence of `$ORIGIN` related to `LD_LIBRARY_PATH` so not sure whether that makes sense (it might do).

Reply:
this is done